.bash_profile

```diff
@@ -1,3 +1,9 @@
+# Prefer GNU coreutils over BSD ones, i.e. ls, cat. This works only if you run 'brew install coreutils', or run brew.sh
+PATH=$(brew --prefix coreutils)/libexec/gnubin:$PATH
+# Allow access to GNU coreutils man pages
+MANPATH=/usr/local/opt/coreutils/libexec/gnuman:${MANPATH}
+export MANPATH
```

Comment:
This might best be wrapped in a condition:

```bash
if [ -d "$(brew --prefix coreutils)/libexec/gnubin" ]; then
    export PATH=$(brew --prefix coreutils)/libexec/gnubin:$PATH
    export MANPATH=$(brew --prefix coreutils)/libexec/gnuman:${MANPATH}
fi
```